### ISSUE TITLE
Consider a more distinct name

### ISSUE THREAD

**Issue by Razzeee** [zwilias/elm-json#17]
Hey there,

I'm not sure if `elm-json` is the ideal name for this. It's very hard to find it via duckduckgo and probably also via google. As there is elm/json and the elm.json files themselves.

A better name might hopefully help to distinguish and market this tool :)

Cheers

**Comment by andys8** [comment]
Same thought. I think it might also be a bit misleading.

**Comment by sporto** [comment]
Agree, someone gave me the link and I thought this had something to do with parsing JSON.

**Comment by matheus23** [comment]
Absolutely. Tried googling it, while looking for it specifically. Google couldn't help much, had to remember 'zwilias'.

Suggestions:
* elm-packager
* elm-solver
* elm-installer
* elm-roots

Just throwing things out there.

**Comment by RalfNorthman** [comment]
To set the expectations really high, it could be named: _elm-cargo_... ;-)
